src/libuast.hpp

```diff
@@ -246,11 +246,17 @@ namespace uast {
             return contexts()[ctx->ctx];
         }
     public:
-        RawContext(NodeRawInterface* iface) {
+        RawContext(NodeRawInterface* iface, Uast* c = nullptr) {
             handle = ++lastHandle();
+            if (c && handle != c->ctx) {
+                std::cerr << "TODO: wrong local handle" << std::endl;
```

Comment:
Is that common/sage to use STDERR in a native library that is to be called from a hosted env like JVM or python?

Reply:
Not safe. This should be fixed before merging it.